### PR TITLE
Database schema

### PR DESCRIPTION
**Pull request by ejepsen88**
SQL file that creates the tables for earthquakes, alerts, networks, types, status and mag_types. 
Also has seeding for typical values for the alerts, networks, types, status, and mag_types tables.

### CODE REVIEW

**Review by joe1606 — CHANGES_REQUESTED** [review]
Overall looks good, just added some small amendments to some details like commas etc. Also I think the order of the table creation may need double checking

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,69 @@
+DROP TABLE earthquakes CASCADE;
+DROP TABLE alerts CASCADE;
+DROP TABLE status CASCADE;
+DROP TABLE mag_types CASCADE;
+DROP TABLE networks CASCADE;
+DROP TABLE types CASCADE;
+
+CREATE TABLE earthquakes (
```
[the remark this answers is not in the document]
Comment (ejepsen88) [reply database/schema.sql:8]: Oh yeah you're right, I'll rearrange them

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    mmi REAL,
+    alert_id SMALLINT NOT NULL,
+    status_id SMALLINT NOT NULL,
+    significance SMALLINT NOT NULL,
+    network_id SMALLINT NOT NULL,
+    nst SMALLINT,
+    dmin REAL,
+    gap REAL,
+    magtype_id SMALLINT NOT NULL,
+    type SMALLINT,
```
[the remark this answers is not in the document]
Comment (ejepsen88) [reply database/schema.sql:25]: Ah yes it should be

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+CREATE TABLE types (
+    type_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    type_value VARCHAR UNIQUE NOT NULL,
+    PRIMARY KEY (type_id)
+);
+
+CREATE TABLE status (
+    status_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    status VARCHAR UNIQUE NOT NULL,
```
Comment (laUmar123) [inline database/schema.sql:28]: I've changed the ERD to be `varchar(9)`

File: database/schema.sql [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    PRIMARY KEY (network_id)
+);
+
+CREATE TABLE types (
+    type_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    type_value VARCHAR UNIQUE NOT NULL,
+    PRIMARY KEY (type_id)
+);
+
+CREATE TABLE status (
```
Comment (laUmar123) [inline database/schema.sql:26]: Also changed the ERD for the table name to be 'statuses' instead of 'status'

File: database/schema.sql [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    status_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    status VARCHAR UNIQUE NOT NULL,
+    PRIMARY KEY (status_id)
+);
+
+CREATE TABLE mag_types (
```
Comment (laUmar123) [inline database/schema.sql:32]: this should be `magtypes`

File: database/schema.sql [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    mag_type_id UNIQUE SMALLINT GENERATED ALWAYS AS IDENTITY,
+    mag_type_value VARCHAR(3) UNIQUE NOT NULL,
+    PRIMARY KEY (mag_type_id)
```
Comment (laUmar123) [inline database/schema.sql:35]: I think it'll be better if all occurrences of 'mag_type' is changed to `magtype_...`

File: database/schema.sql [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    type_id SMALLINT,
+    title TEXT NOT NULL,
+    depth FLOAT,
+    PRIMARY KEY (earthquake_id),
+    FOREIGN KEY (alert_id) REFERENCES alerts(alert_id),
+    FOREIGN KEY (status_id) REFERENCES status(status_id),
+    FOREIGN KEY (network_id) REFERENCES networks(network_id),
+    FOREIGN KEY (type_id) REFERENCES networks(network_id),
+    FOREIGN KEY (mag_type_id) REFERENCES mag_types(mag_type_id)
+);
```
Comment (laUmar123) [inline database/schema.sql:64]: Update to reflect latest ERD

**Review by laUmar123 — CHANGES_REQUESTED** [review]
Nice work, just need to make few updates for latest ERD

File: database/schema.sql (hunk 7 above)
Comment (ejepsen88) [reply database/schema.sql]: Think I've now updated everything so it matches the updated ERD

**Review by laUmar123 — APPROVED** [review]
All good, nice work

**Review by joe1606 — APPROVED** [review]

**Comment by joe1606** [comment]
closes #37

**Pull request merged**